nodebalancer_configs.go

```diff
@@ -103,6 +103,33 @@ type NodeBalancerConfigCreateOptions struct {
 	SSLKey        string           `json:"ssl_key,omitempty"`
 }
 
+// NodeBalancerConfigRebuildOptionsNode for a nodebalancer instance mapping
+type NodeBalancerConfigRebuildOptionsNode struct {
```

Comment:
This is very similar to `NodeBalancerNodeCreateOptions`

https://github.com/linode/linodego/blob/master/nodebalancer_config_nodes.go#L35

Reply:
It was copied from there :). I could not actually verify if the inputs are the same because the new api's do not seem to be in the [Open API spec](https://developers.linode.com/openapi.yaml)